packages/Webkul/Shop/src/Http/Controllers/CartController.php

```diff
@@ -58,6 +56,31 @@ public function index()
         ]);
     }
 
+    /**
+     * Function for get the cart product.
+     *
+     * @return \Illuminate\View\View
+     */
+    public function allProducts()
+    {
+        Cart::collectTotals();
+
+        $cart = Cart::getCart();
+
+        return MiniCart::collection($cart);
+    }
+
+    /**
+     * Function for remove single cart product.
+     *
+     * @return \Illuminate\View\View
```

Comment:
The function does not currently return any value, but based on the comments, it appears that it should return a view.